feature/runtime/exec.go

```diff
@@ -0,0 +1,61 @@
+package runtime
+
+import (
+	"fmt"
+	"github.com/drognisep/systest/feature/runtime/testing"
+	lua "github.com/yuin/gopher-lua"
+	"io/fs"
+	"path/filepath"
+)
+
+func (s *State) newLuaState() *lua.LState {
+	// Lua state initialization should be controlled from here.
+	// https://github.com/yuin/gopher-lua?tab=readme-ov-file#opening-a-subset-of-builtin-modules
+	l := lua.NewState(lua.Options{
+		SkipOpenLibs: true,
+	})
+	// Base lib loading is done here.
```

Comment:
This is how the sandboxing works for this lib. Any Lua modules we don't add into this function will not be available.